pkg/ocm/helpers.go

```diff
@@ -273,3 +273,47 @@ func RandomLabel(size int) string {
 	}
 	return string(chars)
 }
+
+func (c *Client) LinkOrgToRole(orgID string, roleARN string) error {
+	labels, err := c.ocm.AccountsMgmt().V1().Organizations().Organization(orgID).Labels().List().Send()
+	if err != nil {
+		return err
+	}
+	existingARN := ""
+	if labels.Items().Len() > 0 {
+		labels.Items().Range(func(index int, item *amsv1.Label) bool {
+			if item.Key() == "sts_ocm_role" {
+				existingARN = item.Value()
+				return true
+			}
+			return false
+		})
+	}
+	exists := false
+	if existingARN != "" {
+		existingARNArr := strings.Split(existingARN, ",")
+		if len(existingARNArr) > 0 {
+			for _, value := range existingARNArr {
+				if value == roleARN {
+					exists = true
+					break
+				}
+			}
+		}
+	}
+	if exists {
+		return nil
+	}
+	roleARN = existingARN + "," + roleARN
```

Comment:
If there's nothing inside existingARN this creates a leading `,`.

Maybe something like this would be sufficient:
```
if existingARN != "" {
  roleARN = existingARN + "," + roleARN
}
```